tests/ceph_installer/test_ansible.py

```diff
@@ -119,7 +119,16 @@ def run(ceph_cluster, **kw):
     if test_data.get("luns_setting", None) and test_data.get("initiator_setting", None):
         ceph_installer.add_iscsi_settings(test_data)
 
-    LOG.info("Ceph ansible version " + ceph_installer.get_installed_ceph_versions())
+    # Check if ansilbe.log file has any logs initially
+    if custom_log:
+        ansible_log_file, _ = ceph_installer.exec_command(
+            cmd="cat {}/ansible.cfg | grep log_path | cut -d \" \" -f3 | tr -d '\n\r'".format(
+                ansible_dir
+            )
+        )
+        file_size_before, _ = ceph_installer.exec_command(
+            cmd='ls -l "{}" | cut -d " " -f5 | tr -d \'\n\r\''.format(ansible_log_file)
```

Comment:
Could we use the command `du`?

Reply:
@psathyan, I was thinking the same previously! But, `ls` looks more simple to me and gives more options if we'll need to modify it in future. Let me know if you still want me to use `du`.

Reply:
Here the requirement is to check the size of the file (existing) hence would prefer using `du` as it is straight forward.

Reply:
Sure, will use `du` instead.

Reply:
Updated the changes.